next.js/app/BUILD.bazel

```diff
@@ -0,0 +1,41 @@
+load("@aspect_rules_js//js:defs.bzl", "js_test")
+load("@npm//:defs.bzl", "npm_link_all_packages")
+load("//bazel:next.bzl", "next")
+
+npm_link_all_packages(name = "node_modules")
+
+# Run `bazel build //app:next` to build (equivalent to running `next build`).
+#
+# It is recommended that you use [ibazel](https://github.com/bazelbuild/bazel-watcher)
+# to run the devserver targets.
+#
+# Run `ibazel run //app:next_dev` to run the development server in watch mode
+# (equivalent to running `next dev`).
+#
+# Run `ibazel run //app:next_start` to run the production server in watch mode
+# (equivalent to running `next start`).
+next(
+    name = "next",
+    srcs = [
+        "//app/pages",
+        "//app/public",
+        "//app/styles",
+    ],
+    data = [
+        "next.config.js",
+        ":node_modules/next",
+        ":node_modules/react",
+        ":node_modules/react-dom",
+        ":package.json",
+        "//:node_modules/typescript",
+    ],
+    next_bin = "./node_modules/.bin/next",
```

Comment:
I guess a given BUILD will have it's own nextjs so we can't put this attribute into the macro?